src/platform/platform_winuser.c

```diff
@@ -69,30 +68,30 @@ CxPlatSystemUnload(
 
 _IRQL_requires_max_(PASSIVE_LEVEL)
 _Must_inspect_result_
-_Success_(return != FALSE)
-BOOLEAN
+QUIC_STATUS
 CxPlatProcessorGroupInfo(
     _In_ LOGICAL_PROCESSOR_RELATIONSHIP Relationship,
     _Outptr_ _At_(*Buffer, __drv_allocatesMem(Mem)) _Pre_defensive_
         PSYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX* Buffer,
     _Out_ PDWORD BufferLength
     );
 
-BOOLEAN
+QUIC_STATUS
 CxPlatProcessorInfoInit(
     void
     )
 {
-    BOOLEAN Result = FALSE;
+    QUIC_STATUS Status = QUIC_STATUS_SUCCESS;
     DWORD BufferLength = 0;
     uint8_t* Buffer = NULL;
     uint32_t Offset;
 
-    uint32_t ActiveProcessorCount = CxPlatProcActiveCount();
+    const uint32_t ActiveProcessorCount = CxPlatProcActiveCount();
     uint32_t ProcessorGroupCount = 0;
     uint32_t ProcessorsPerGroup = 0;
-    uint32_t NumaNodeCount = 0;
 
+    CXPLAT_DBG_ASSERT(ActiveProcessorCount > 0);
```

Comment:
Should return error when `ActiveProcessorCount == 0`.

See: https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getactiveprocessorcount